ecosystem/python/sdk/Makefile

```diff
@@ -6,8 +6,8 @@ test:
 
 fmt:
 	find ./examples ./aptos_sdk *.py -type f -name "*.py" | xargs poetry run autoflake -i -r --remove-all-unused-imports --remove-unused-variables --ignore-init-module-imports
-	poetry run black aptos_sdk examples setup.py
-	poetry run isort aptos_sdk examples setup.py
+	- poetry run isort aptos_sdk examples setup.py
+	- poetry run black aptos_sdk examples setup.py
```

Comment:
why? `-` means ignore failures
I don't care about the ordering.

Reply:
Because isort and black enforce different coding styles and they will undo each other's changes. Isort returns 1 if any file was reformatted, indicating an error, but if black reformats the code back after it, it doesn't mean anything. It's a spurious error.
The real error checking is done in lint-test.yaml: if after running all the formatters, there are modified files locally, it's an error, otherwise pass.

Reply:
that's not accurate... if there are syntax issues, isort and black will fail to run and then the linter will pass becaues there are no changes.

Reply:
Right now isort and black are *always* giving false positives, so their use is kind of pointless.

Reply:
Ideally we should be able to ask both tools ignore reformatted files and only throw errors for more serious conditions.